prashant/HTML/Assignment5/aboutme.css

```diff
@@ -0,0 +1,31 @@
+body{
```

Comment:
after body tag you need to give one space

Reply:
for every css tag

Reply:
ok sir now i have done it please cheack